db/migrate/20190206223630_create_icons.rb

```diff
@@ -0,0 +1,13 @@
+class CreateIcons < ActiveRecord::Migration[5.2]
+  def change
+    create_table :icons do |t|
```

Comment:
I realize this is just the migration, but I am wondering if will we be using this new table yet or can this be moved to a later PR when we support overriding the image from the Platform?

Note: When this table does get added you will also need a `tenant_id` column.

Reply:
Yep I was getting ahead of myself! Saving this for later.